libraries/hotstuff/include/eosio/hotstuff/hotstuff.hpp

```diff
@@ -4,18 +4,17 @@
 #include <fc/crypto/bls_private_key.hpp>
 #include <fc/crypto/bls_public_key.hpp>
 #include <fc/crypto/bls_signature.hpp>
-#include <fc/crypto/bls_utils.hpp>
 
 #include <boost/dynamic_bitset.hpp>
 
-namespace eosio::chain {
+namespace eosio::hotstuff {
```

Comment:
Why do we even have a `eosio::hotstuff` namespace? I think it would be better to have it all live under `eosio::chain`.  Many types already start with `hs_` so there is no ambiguity. And things like `quorum_certificate` are not ambiguous either. I even wonder why we have a separate directory and  library for hotstuff.

Reply:
The `hotstuff` namespace exists and I do think having this namespace and directory is helpful. Since it does exist, I believe these types belong in the namespace.

Reply:
I agree we probably don't need a separate directory for hotstuff, especially now that pacemaker is owned by controller. But we can do those move around later after prototyping.